_inc/client/components/form/form-toggle/index.jsx

```diff
@@ -5,9 +5,14 @@
  * External dependencies
  */
 import PropTypes from 'prop-types';
-import React, { Component } from 'react';
+import React, { Component, Fragment } from 'react';
 import classNames from 'classnames';
 
+/**
+ * External dependencies
```

Comment:
This tag seems to be duplicate. Did you mean to make it `WordPress dependencies` or perhaps it should be part of the above block of imports?